src/pudl/metadata/fields.py

```diff
@@ -3912,6 +3981,14 @@
         "description": "Estimated electricity demand scaled by the total sales within a state.",
         "unit": "MWh",
     },
+    "sec_company_id": {
+        "type": "string",
+        "description": "Assigned identifier for the company.",
+    },
```

Comment:
As with the IRS ID above, according to our naming conventions I think this would be `company_id_sec` Description should mention SEC.

Reply:
Fixed.